index.js

```diff
@@ -62,10 +62,14 @@ BeamAccessory.prototype.off = function() {
   }).then(client => {
     return wait(long_timeout, client)
   }).then(client => {
-    return writeTo(client, "screen;0;0")
+    return wait(timeout, client)
+  }).then(client => {
+    return writeTo(client, "led;3;3")
```

Comment:
Hack to turn things off is to turn on the light, which will turn off the projector, then turn off the light which turns off the light, leaving the projector also off.